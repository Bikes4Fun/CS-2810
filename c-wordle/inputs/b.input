si(l)ly
a[l]o[e](d)
[e][l][d][e][r]
