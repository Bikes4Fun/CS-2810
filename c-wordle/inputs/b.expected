viable candidate: rebus
viable candidate: resus
viable candidate: rubes
viable candidate: rudes
viable candidate: ruers
viable candidate: rules
viable candidate: rumes
viable candidate: runes
viable candidate: ruses
